go/libkb/resolve.go

```diff
@@ -99,7 +96,7 @@ func resolveUsername(au AssertionURL) (res ResolveResult) {
 	ha := HTTPArgsFromKeyValuePair(key, S{val})
 	ha.Add("multi", I{1})
 	ha.Add("fields", S{"basics,public_keys,pictures"})
-	ares, res.err = G.API.Get(APIArg{
+	ares, res.err = r.G().API.Get(APIArg{
```

Comment:
Is it possible to reduce the `fields` for resolve? Would it be measurably faster with just `basics` (if that's all that's needed)?